src/sdl2/hint.rs

```diff
@@ -3,6 +3,27 @@ use sys::hint as ll;
 use std::ptr;
 use libc::c_char;
 
+const VIDEO_MINIMIZE_ON_FOCUS_LOST: &'static str = "SDL_VIDEO_MINIMIZE_ON_FOCUS_LOST";
+
+/// Sets the SDL_HINT_VIDEO_MINIMIZE_ON_FOCUS_LOSS hint.
```

Comment:
How about `A hint that specifies if a SDL_Window is minimized if it loses key focus when in fullscreen mode.` (directly from sdl's docs)?